connection.js

```diff
@@ -17,6 +20,10 @@ import { setPrejoinDisplayNameRequired } from './react/features/prejoin/actions'
 
 const logger = Logger.getLogger(__filename);
 
+const isTokenAuthEnabled
```

Comment:
Since you are moving this code around, can you compute it dynamically (wrapped in a function for example) instead of accessing the `config` global? You can get the store with `APP.store.getState()`.